content/zh/docs/concepts/architecture/nodes.md

```diff
@@ -124,38 +319,40 @@ The node condition is represented as a JSON object. For example, the following r
-与此同时，被计划删除的 Pods 可能会继续在游离的节点上运行。
+某些情况下，当节点不可达时，API 服务器不能和其上的 kubelet 通信。
+删除 Pod 的决定不能传达给 kubelet，直到它重新建立和 API 服务器的连接为止。
+与此同时，被计划删除的 Pod 可能会继续在游离的节点上运行。
 
 <!--
-In versions of Kubernetes prior to 1.5, the node controller would [force delete](/docs/concepts/workloads/pods/pod/#force-deletion-of-pods)
-these unreachable pods from the apiserver. However, in 1.5 and higher, the node controller does not force delete pods until it is
-confirmed that they have stopped running in the cluster. You can see the pods that might be running on an unreachable node as being in
-the `Terminating` or `Unknown` state. In cases where Kubernetes cannot deduce from the underlying infrastructure if a node has
-permanently left a cluster, the cluster administrator may need to delete the node object by hand.  Deleting the node object from
-Kubernetes causes all the Pod objects running on the node to be deleted from the apiserver, and frees up their names.
+The node controller does not force delete pods until it is confirmed that they have stopped
+running in the cluster. You can see the pods that might be running on an unreachable node as
+being in the `Terminating` or `Unknown` state. In cases where Kubernetes cannot deduce from the
+underlying infrastructure if a node has permanently left a cluster, the cluster administrator
+may need to delete the node object by hand.  Deleting the node object from Kubernetes causes
+all the Pod objects running on the node to be deleted from the API server, and frees up their
+names.
 -->
-在 1.5 版本之前的 Kubernetes 上，节点控制器会将不能访问的 Pods 从 apiserver 中
-[强制删除](/docs/concepts/workloads/pods/pod/#force-deletion-of-pods)。
-但在 1.5 或更高的版本里，在节点控制器确认这些 Pods 在集群中已经停止运行前，不会强制删除它们。
-你可以看到这些可能在无法访问的节点上运行的 Pods 处于 `Terminating` 或者 `Unknown` 状态。
-如果 kubernetes 不能基于下层基础设施推断出某节点是否已经永久离开了集群，集群管理员可能需要手动删除该节点对象。
-从 Kubernetes 删除节点对象将导致 apiserver 删除节点上所有运行的 Pod 对象并释放它们的名字。
+节点控制器在确认 Pod 在集群中已经停止运行前，不会强制删除它们。
+你可以看到这些可能在无法访问的节点上运行的 Pod 处于 `Terminating` 或者 `Unknown` 状态。
+如果 kubernetes 不能基于下层基础设施推断出某节点是否已经永久离开了集群，
+集群管理员可能需要手动删除该节点对象。
+从 Kubernetes 删除节点对象将导致 API 服务器删除节点上所有运行的 Pod 对象并释放它们的名字。
 
 <!--
 The node lifecycle controller automatically creates
-[taints](/docs/concepts/configuration/taint-and-toleration/) that represent conditions.
-When the scheduler is assigning a Pod to a Node, the scheduler takes the Node's taints
-into account, except for any taints that the Pod tolerates.
+[taints](/docs/concepts/scheduling-eviction/taint-and-toleration/) that represent conditions.
+The scheduler takes the Node's taints into consideration when assigning a Pod to a Node.
+Pods can also have tolerations which let them tolerate a Node's taints.
 -->
-节点生命周期控制器会自动创建代表条件的[污点](/docs/concepts/configuration/taint-and-toleration/)。
-当调度器将 Pod 指派给某节点时，调度器会考虑节点上的污点，但是 Pod 可以容忍的污点除外。
+节点生命周期控制器会自动创建代表状况的
+[污点](/zh/docs/concepts/scheduling-eviction/taint-and-toleration/)。
+当调度器将 Pod 指派给某节点时，会考虑节点上的污点。
+Pod 则可以通过容忍度（Toleration）表达所能容忍的污点。
```

Comment:
```suggestion
Pod 则可以通过容忍度（toleration）表达所能容忍的污点。
```

(to emphasize that toleration is _not_ the name of an API resource).

Reply:
The convention for translation in Chinese is to have the English term added in brace after the translated version. First letter is upper case.